lib/models/pricing_phase_wrapper.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+import 'period_wrapper.dart';
+import 'price_wrapper.dart';
+
+part 'pricing_phase_wrapper.freezed.dart';
+part 'pricing_phase_wrapper.g.dart';
+
+@freezed
+
+/// Contains all the details associated with a PricingPhase
+class PricingPhase with _$PricingPhase {
+  const factory PricingPhase(
+    /// Billing period for which the PricingPhase applies
+    @JsonKey(name: 'billingPeriod') Period billingPeriod,
```

Comment:
Can this be null? I see phc is doing         `"billingPeriod" to billingPeriod?.mapPeriod()`

Reply:
Ahhhh, it probably can be! Will fix 💪